src/disasm.rs

```diff
@@ -17,7 +17,6 @@ pub struct DisasmOpts {
 
 #[derive(Debug, Clone)]
 struct DisasmExtra {
-  pub opts: DisasmOpts,
```

Comment:
Probably keep this since opts will be needed to render addresses differently depending on which flags were used. I hadn't tested the other options (just my current ones).